src/module.indexer/model/dftx/place.auction.bid.ts

```diff
@@ -12,25 +13,43 @@ export class PlaceAuctionBidIndexer extends DfTxIndexer<PlaceAuctionBid> {
   private readonly logger = new Logger(PlaceAuctionBidIndexer.name)
 
   constructor (
-    private readonly vaultAuctionHistoryMapper: VaultAuctionHistoryMapper
+    private readonly vaultAuctionHistoryMapper: VaultAuctionHistoryMapper,
+    private readonly vaultAuctionBatchBidMapper: VaultAuctionBatchBidMapper
   ) {
     super()
   }
 
   async indexTransaction (block: RawBlock, transaction: DfTxTransaction<PlaceAuctionBid>): Promise<void> {
     const data = transaction.dftx.data
+    const from = toBuffer(data.from.stack).toString('hex')
 
     await this.vaultAuctionHistoryMapper.put({
       id: `${data.vaultId}-${data.index}-${transaction.txn.txid}`,
       key: `${data.vaultId}-${data.index}`,
       sort: `${HexEncoder.encodeHeight(block.height)}-${transaction.txn.txid}`,
       vaultId: data.vaultId,
       index: data.index,
-      from: toBuffer(data.from.stack).toString('hex'),
+      from: from,
       amount: data.tokenAmount.amount.toString(),
       tokenId: data.tokenAmount.token,
       block: { hash: block.hash, height: block.height, medianTime: block.mediantime, time: block.time }
     })
+
+    // TODO(canonbrother): due to the reason WIP on loanScheme and loan ops indexing
+    // currently provides a temp solution first
+    // FE requirement is to label, eg: Bid Lost, on auction listing page
+    const batchBid = await this.vaultAuctionBatchBidMapper.get(`${data.vaultId}-${data.index}`)
```

Comment:
```suggestion
    const batchBid = await this.vaultAuctionBatchBidMapper.get(`${data.vaultId}-${auction-close-timestamp}-${data.index}`)
```

eg: 21st Jan liquidated, auctioned, collateral partially bided and back to active after 6 hours. 22nd Jan get liquidated again, same `${vaultId}-${batch-zero let say}`

Reply:
that's no liquidationHeight for me
here's the fixes
https://github.com/DeFiCh/whale/pull/698#discussion_r810940543